<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::AppointmentNew"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E1E2F;
border-color: rgb(116, 248, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1E1E2F;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackWidget1">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="Page1">
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,0">
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(205, 221, 255);</string>
          </property>
          <property name="text">
           <string>Camera Status:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextBrowser {
    background-color: #1E1E2F;
}

QScrollBar:vertical {
    width: 12px;
    background: #f0f0f0;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #3498db;
    min-height: 20px;
    border-radius: 6px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}





</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="styleSheet">
           <string notr="true">    QGroupBox {
        font-size: 16px;
        font-weight: bold;
        color: rgb(205, 221, 255); /* 标题颜色 */
        border: 1px solid gray;
        margin-top: 20px; /* 标题与边框距离 */
    }

    QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top center; /* 标题居中 */
        padding: 0 3px;
        background-color: transparent;
    }
</string>
          </property>
          <property name="title">
           <string>Current Info View</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(205, 221, 255);</string>
          </property>
          <property name="text">
           <string>Message Display</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(213, 245, 255);</string>
            </property>
            <property name="text">
             <string>CONNECT</string>
            </property>
            <property name="autoRepeatDelay">
             <number>304</number>
            </property>
            <property name="autoRepeatInterval">
             <number>97</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string extracomment="IP"/>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(219, 254, 255);</string>
            </property>
            <property name="placeholderText">
             <string>ip,rack,slot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 246, 255);</string>
            </property>
            <property name="text">
             <string>DISCONNECT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(217, 249, 255);</string>
            </property>
            <property name="text">
             <string>READ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(212, 245, 255);</string>
            </property>
            <property name="placeholderText">
             <string>DB,StartAddress,ReadLengh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(211, 241, 255);</string>
            </property>
            <property name="text">
             <string>Write</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="styleSheet">
             <string notr="true">color: rgb(226, 248, 255);</string>
            </property>
            <property name="placeholderText">
             <string>DB,StartAddress</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="styleSheet">
             <string notr="true">color: rgb(222, 246, 255);</string>
            </property>
            <property name="placeholderText">
             <string>DATA</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">    QGroupBox {
        font-size: 16px;
        font-weight: bold;
        color: rgb(205, 221, 255); /* 标题颜色 */
        border: 1px solid gray;
        margin-top: 20px; /* 标题与边框距离 */
    }

    QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top center; /* 标题居中 */
        padding: 0 3px;
        background-color: transparent;
    }
</string>
          </property>
          <property name="title">
           <string>Camera</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QVideoWidget" name="cameraWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="styleSheet">
        <string notr="true">background-color: #1E1E2F;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1" columnstretch="0,0">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="styleSheet">
             <string notr="true">    QGroupBox {
        font-size: 16px;
        font-weight: bold;
        color: rgb(205, 221, 255); /* 标题颜色 */
        border: 1px solid gray;
        margin-top: 20px; /* 标题与边框距离 */
    }

    QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top center; /* 标题居中 */
        padding: 0 3px;
        background-color: transparent;
    }
</string>
            </property>
            <property name="title">
             <string>IP,Port Used</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QTableWidget" name="tableWidget_2">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="styleSheet">
             <string notr="true">    QGroupBox {
        font-size: 16px;
        font-weight: bold;
        color: rgb(205, 221, 255); /* 标题颜色 */
        border: 1px solid gray;
        margin-top: 20px; /* 标题与边框距离 */
    }

    QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top center; /* 标题居中 */
        padding: 0 3px;
        background-color: transparent;
    }
</string>
            </property>
            <property name="title">
             <string>Disk usage</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QTableWidget" name="tableWidget_3"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCustomPlot" name="customPlot" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: #1E1E2F;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_9" rowstretch="0">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8" rowstretch="1,9,1">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(205, 221, 255);</string>
            </property>
            <property name="text">
             <string>Param setting</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="tableWidget_4"/>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,9">
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(211, 241, 255),   /* 顶部颜色 */
        stop:1 rgb(150, 200, 230)    /* 底部颜色 */
    );
    color: black;                /* 字体颜色 */
    border: none;                /* 去掉边框线 */
    border-radius: 10px;         /* 圆角半径 */
    padding: 6px 12px;           /* 内边距 */
    font-size: 14px;
}
QPushButton:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(190, 230, 250),
        stop:1 rgb(130, 180, 210)
    );  /* 悬停颜色 */
}
QPushButton:pressed {
       background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1, 
        stop:0 #141420, 
        stop:1 #2E2E48
    );   /* 按下颜色 */
}
</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(211, 241, 255),   /* 顶部颜色 */
        stop:1 rgb(150, 200, 230)    /* 底部颜色 */
    );
    color: black;                /* 字体颜色 */
    border: none;                /* 去掉边框线 */
    border-radius: 10px;         /* 圆角半径 */
    padding: 6px 12px;           /* 内边距 */
    font-size: 14px;
}
QPushButton:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(190, 230, 250),
        stop:1 rgb(130, 180, 210)
    );  /* 悬停颜色 */
}
QPushButton:pressed {
       background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1, 
        stop:0 #141420, 
        stop:1 #2E2E48
    );   /* 按下颜色 */
}
</string>
              </property>
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_10" rowstretch="1,9">
        <item row="1" column="0">
         <widget class="QTableView" name="tableView">
          <property name="styleSheet">
           <string notr="true">/* 整体表格 */
QTableView {
    background-color: #1E1E2F; /* 深色背景 */
    color: #E0E0E0;           /* 默认文字颜色 */
    gridline-color: #2C2C3C;  /* 网格线颜色 */
    border: 1px solid #00BFFF; /* 科技蓝边框 */
    selection-background-color: #007ACC; /* 选中背景 */
    selection-color: white;   /* 选中字体颜色 */
    alternate-background-color: #252538; /* 交替行颜色 */
    font-size: 14px;
    font-family: &quot;Microsoft YaHei&quot;;
}

/* 表头 */
QHeaderView::section {
    background-color: #0D1B2A;
    color: #00BFFF;
    font-weight: bold;
    font-size: 14px;
    padding: 4px;
    border: 1px solid #1B263B;
}

/* 鼠标悬停单元格 */
QTableView::item:hover {
    background-color: rgba(0, 191, 255, 0.2);
}

/* 鼠标悬停且选中 */
QTableView::item:selected:hover {
    background-color: #0096FF;
}

/* 编辑状态下的输入框（QLineEdit） */
QTableView QLineEdit {
    background-color: #0D1B2A; /* 深色背景 */
    color: #00FFEA;           /* 科技蓝绿色字体 */
    border: 1px solid #00BFFF;
    selection-background-color: #007ACC;
    selection-color: white;
}
QTableCornerButton::section {
    border: none;
background-color: #0D1B2A;
}
</string>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1,1,1,9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">
  color: rgb(205, 221, 255);               /* 字体颜色 */
    border: none;                /* 去掉边框线 */
    border-radius: 10px;         /* 圆角半径 */
    padding: 9px 0px;           /* 内边距 */
    font-size: 14px;


</string>
            </property>
            <property name="text">
             <string>User:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(211, 241, 255),   /* 顶部颜色 */
        stop:1 rgb(150, 200, 230)    /* 底部颜色 */
    );
    color: black;                /* 字体颜色 */
    border: none;                /* 去掉边框线 */
    border-radius: 10px;         /* 圆角半径 */
    padding: 6px 12px;           /* 内边距 */
    font-size: 14px;
}
QPushButton:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(190, 230, 250),
        stop:1 rgb(130, 180, 210)
    );  /* 悬停颜色 */
}
QPushButton:pressed {
       background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1, 
        stop:0 #141420, 
        stop:1 #2E2E48
    );   /* 按下颜色 */
}
</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(211, 241, 255),   /* 顶部颜色 */
        stop:1 rgb(150, 200, 230)    /* 底部颜色 */
    );
    color: black;                /* 字体颜色 */
    border: none;                /* 去掉边框线 */
    border-radius: 10px;         /* 圆角半径 */
    padding: 6px 12px;           /* 内边距 */
    font-size: 14px;
}
QPushButton:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(190, 230, 250),
        stop:1 rgb(130, 180, 210)
    );  /* 悬停颜色 */
}
QPushButton:pressed {
       background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1, 
        stop:0 #141420, 
        stop:1 #2E2E48
    );   /* 按下颜色 */
}
</string>
            </property>
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(211, 241, 255),   /* 顶部颜色 */
        stop:1 rgb(150, 200, 230)    /* 底部颜色 */
    );
    color: black;                /* 字体颜色 */
    border: none;                /* 去掉边框线 */
    border-radius: 10px;         /* 圆角半径 */
    padding: 6px 12px;           /* 内边距 */
    font-size: 14px;
}
QPushButton:hover {
    background-color: qlineargradient(
        spread:pad,
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(190, 230, 250),
        stop:1 rgb(130, 180, 210)
    );  /* 悬停颜色 */
}
QPushButton:pressed {
       background-color: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:0, y2:1, 
        stop:0 #141420, 
        stop:1 #2E2E48
    );   /* 按下颜色 */
}
</string>
            </property>
            <property name="text">
             <string>Export</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QMenuBar {
        background-color: #1E1E2F;
        color: white;
    }

    QMenuBar::item {
        background-color: transparent;
        padding: 5px 15px;
    }

    QMenuBar::item:selected {
        background-color: darkCyan;  /* 鼠标悬停时的颜色 */
        color: white;
    }

    QMenuBar::item:pressed {
        background-color: #007ACC;  /* 鼠标点击时的颜色 */
    }</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">        QMenu {
            background-color: #1E1E2F;
            color: white;
            border: 1px solid #444;
        }

        QMenu::item {
            background-color: transparent;
            padding: 6px 20px;
        }

        QMenu::item:selected {
            background-color: darkCyan;
            color: white;
        }

        QMenu::item:pressed {
            background-color: #007ACC;
        }</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionac1_2"/>
    <addaction name="separator"/>
    <addaction name="actionac2_2"/>
    <addaction name="separator"/>
    <addaction name="actionac1_3"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionac1">
   <property name="text">
    <string>ac1</string>
   </property>
  </action>
  <action name="actionac2">
   <property name="text">
    <string>ac2</string>
   </property>
  </action>
  <action name="actionac1_2">
   <property name="text">
    <string>Main</string>
   </property>
  </action>
  <action name="actionac2_2">
   <property name="text">
    <string>System monitor</string>
   </property>
  </action>
  <action name="actionac1_3">
   <property name="text">
    <string>Param</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
